config/client.yaml

```diff
@@ -0,0 +1,56 @@
+config:
+  branches:
+    release-v1.15:
+      openShiftVersions:
+        - version: "4.15"
+          skipCron: true
+repositories:
+  - dockerfiles: {}
+    e2e:
+      - match: .*e2e$
+        skipCron: true
+        skipImages:
+            - knative-client-client
+            - kn-cli-artifacts
+    ignoreConfigs:
+      matches:
+        - .*release-next*
+        - .*release-v1.*.yaml$
+    customConfigs:
+      - releaseBuildConfiguration:
+          binary_build_commands: |
+            make install
+            make build-cross
+          test_binary_build_commands: make test install
```

Comment:
This's supported by the change in `prowgen_config.go` from L263. 

So far I've added a generated Dockerfiles to client `release-v1.15`. We used this build binary mechanism since the forever. See the custom Dockerfile.cliartifacts and the injection. 

I'll look into reworking the image to have its own build phase, but I'm not quiet sure how to wire it with generator as it's additional image definition.

Reply:
cc @mgencur